lib/language.js

```diff
@@ -2500,6 +2500,18 @@ function init() {
       ,hr: 'Vrijednost GUK-a'
       ,it: 'Lettura glicemie'
       }
+      ,'Announcement' : {
+      cs: ''
```

Comment:
Shouldn't set the other language to `''`, that will cause a empty values instead of falling back to English